yarn.lock

```diff
@@ -4236,7 +4236,7 @@
   dependencies:
     "@octokit/openapi-types" "^5.3.2"
 
-"@octokit/types@^6.13.0":
+"@octokit/types@^6.13.0", "@octokit/types@^6.8.2":
```

Comment:
Would it be possible to do this in another PR if it's needed at all? :)

Reply:
I think this is just a yarn lock drift. It's fixed in another PR. So gona merge :)

Reply:
yeah sorry for the delay, this just happened when I ran `yarn install`